libraries/ui/src/ui/ToolbarScriptingInterface.cpp

```diff
@@ -10,31 +10,31 @@
 
 #include <QtCore/QThread>
 #include <QtQuick/QQuickItem>
-#include <QtScript/QScriptValue>
-#include <QtScript/QScriptEngine>
+#include <ScriptValue.h>
+#include <ScriptEngine.h>
 
 #include <shared/QtHelpers.h>
 #include "../OffscreenUi.h"
 
-QScriptValue toolbarToScriptValue(QScriptEngine* engine, ToolbarProxy* const &in) {
+ScriptValue toolbarToScriptValue(ScriptEngine* engine, ToolbarProxy* const &in) {
     if (!in) {
         return engine->undefinedValue();
     }
-    return engine->newQObject(in, QScriptEngine::QtOwnership, QScriptEngine::ExcludeDeleteLater | QScriptEngine::ExcludeChildObjects);
```

Comment:
I ended up making these properties isolated to the private QtScript interface.  The current default on that private interface is actually this exact value, but I did *change* that default (I think I may have added ExcludeChildObjects) so functions calling newQObject *without* the second parameter may have slightly different behavior.  This was potentially-controversial enough that I did a few code-review passes in case it was likely to break something, but our use of this second parameter suggests this isn't something I'm expecting to code in the V8 implementation.